theme/views/@layout.latte

```diff
@@ -19,13 +21,26 @@
 	{$mangoSnippet}
 	{initComponent Shapes, [ data => [ url => "{$assetsPath}/sprites/{$buildstamp}shapes.svg" ] ]}
 
-	{block body}{/block}
+	{block body}
+	<div class="page">
+		{block alerts}
+			<div class="alerts" n:if="!empty($flashes)">
+				<div n:foreach="$flashes as $flash" class="alert type-{$flash->type}">{$flash->message}</div>
+			</div>
+		{/block}
+		{block page}{/block}
+	</div>
+	{/block}
 
 	{php wp_footer()}
 
 	{block scripts}
 		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js" defer></script>
+		<script src="https://cdn.jsdelivr.net/npm/nette-forms@3.0.2/src/assets/netteForms.min.js" defer></script>
```

Comment:
Obecně jsem proti plošnému použití `netteForms` všude. Inicializace tohotok je stále velmi náročná na resources, děje se tak všude na každé stránce (i když není potřeba). Proč to nemáme v samostatné komponentě?